emission/individual_tests/TestNominatim.py

```diff
@@ -0,0 +1,113 @@
+from __future__ import unicode_literals
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+from future import standard_library
+standard_library.install_aliases()
+from builtins import *
+import unittest
+import unittest.mock as mock
+# import overpy
+import os
+from emission.core.wrapper.trip_old import Coordinate
+import requests
+import emission.core.wrapper.entry as ecwe
+import emission.core.wrapper.wrapperbase as ecww
+import emission.net.ext_service.geocoder.nominatim as eco
+import emission.analysis.intake.cleaning.clean_and_resample as clean
+
+#temporarily sets NOMINATIM_QUERY_URL to the environment variable for testing.
+NOMINATIM_QUERY_URL_env = os.environ.get("NOMINATIM_QUERY_URL")
+
+NOMINATIM_QUERY_URL = (
+    NOMINATIM_QUERY_URL_env if NOMINATIM_QUERY_URL_env is not None 
+    else eco.NOMINATIM_QUERY_URL
+)
+
+GFBK = os.environ.get("GFBK_KEY")
+
+GEOFABRIK_QUERY_URL = (
+    "https://geocoding.geofabrik.de/{}".format(GFBK) if GFBK is not None
+    else print("No key available")
+)
+
+#Creates a fake place in Rhode Island to use for testing.
+fake_id = "rhodeislander"
+key = "segmentation/raw_place"
+write_ts = 1694344333
+data = {'source': 'FakeTripGenerator','location': {'type': 'Point', 'coordinates': [-71.4128343, 41.8239891]}}
+fake_place = ecwe.Entry.create_fake_entry(fake_id, key, data, write_ts)
+
+class NominatimTest(unittest.TestCase):
+    maxDiff = None
+
+    #Basic query to check that nominatim and geofabrik are calling the same area.
+    def test_geofabrik_and_nominatim(self):
+        nominatim_result = requests.get(NOMINATIM_QUERY_URL + "/reverse?lat=41.8239891&lon=-71.4128343&format=json").json()
+        geofabrik_result = requests.get(GEOFABRIK_QUERY_URL + "/reverse?lat=41.8239891&lon=-71.4128343&format=json").json()
```

Comment:
this is the only place where you use the `GEOFABRIK_QUERY_URL`. All other calls use nominatim OSM. This is not consistent with our prior plan: https://github.com/e-mission/e-mission-docs/issues/937#issuecomment-1716280421

Can you elaborate on the reasons for the change?

Reply:
For the tests and geofabrik to work in production, we'll have to change the OPENSTREETMAP_QUERY_URL in nominatim.py to use the geofabrik URL with the API key. Currently, it passes in the url as an environment variable (which is the container URL during testing), but defaults to the nominiatim url if an env isn't set. We'll have to pass in the geofabrik url + API key as an environment variable with AWS secrets manager (or whatever method you had in mind) for production. Otherwise, the calls in the test are made to the container. I think I am confused about what you are suggesting!

Reply:
@shankari Any comments or suggestions?

Reply:
Okay, I think you mean something like this:
```
imports .....
def nominatim(nominatim_query_url):
    results = requests.get(nominatim_query_url)
    return results.json()

def test(self):
    lat = "41.8239891"
    lon = '-71.4128343'
    osm_query_url = f"https://nominatim.openstreetmap.org/reverse?lat={lat}&lon={lon}&format=json"
    geofabrik_query_url = f"https://geofabrik.de/...reverse?lat={lat}&lon={lon}&format=json"
    geofabrik_result = nominatim(geofabrik_query_url)
    osm_result = nominatim(osm_query_url)
    self.assertequal(geofabrik_result, osm_result)
```
Is that what you're suggesting?

Reply:
yup. although I think that the URL Is not an argument - it is passed in as an environment variable.
So you will need to set the environment variable and then call `make_json_reverse` or equivalent on the module